src/OmniSharp.Roslyn/Services/AbstractCodeActionProvider.cs

```diff
@@ -30,31 +32,21 @@ protected AbstractCodeActionProvider(string providerName, ImmutableArray<Assembl
 
             this.CodeRefactoringProviders = types
```

Comment:
hmm.. is this different one that the other one that got code fix provider from analyzer assembly loader? is it one for built in roslyn analyzer?